### DISCUSSION TITLE
Problem with probing in Eddy3D

### DISCUSSION THREAD

**Discussion by Soin275** [Eddy3D-Dev/Discussions#5] [Q&A]
I've tried to make the test file "SimpleWindAnalysisiWithTerrain" to work but I have some problems. After running the simulation and the probing it looks like everything will work but I can't see any vectors in Rhino, only points. When I look at the probing component I see this (image1). What can the problem be and how do I fix it? I use Rhino 8.13.24317.13001, Eddy3D 4.8.0 and BlueCDF-Core-2020
![Image1](https://github.com/user-attachments/assets/2105a7be-7716-4c6d-ae22-78a690f9f48c)
![image2](https://github.com/user-attachments/assets/d18e804a-a2ff-4a00-9de4-3a4ed1a95c5c)

**Comment by kastnerp** [comment]
[3_SimpleWindAnalysisWithTerrain.ghx.zip](https://github.com/user-attachments/files/17878443/3_SimpleWindAnalysisWithTerrain.ghx.zip)
Could you try this and let me know if it works?

**Reply by Soin275** [reply]
I've tried your file but it wont work, I still only get points
![Image1](https://github.com/user-attachments/assets/6d0853e6-caec-4836-99e5-6843d25f0a11)
![image2](https://github.com/user-attachments/assets/23b619d2-84e3-4bbb-9810-38da013910e0)

**Reply by kastnerp** [reply]
Please make sure you have a simulation that ran successfully. I'd suggest to check with Paraview first. See [here](https://www.eddy3d.com/support/documentation/) for some examples.
